addons/mail/static/src/messaging/entity/chat_window_manager/chat_window_manager.js

```diff
@@ -420,6 +421,9 @@ function ChatWindowManagerFactory({ Entity }) {
             compute: '_computeLastVisible',
             dependencies: ['allOrderedVisible'],
         }),
+        messaging: many2one('Messaging', {
+            compute: '_computeMessaging',
```

Comment:
This should not be a compute I think? It's just the inverse of `chatWindowManager`.

But, it's actually convenient to have this a compute/default so that it is immediately set during create, instead of having to wait for the link to be done.

Is there a way to reconcile the two?